shell/platform/darwin/macos/framework/Source/FlutterMutatorView.mm

```diff
@@ -26,6 +26,20 @@ @interface FlutterMutatorView () {
 
 @end
 
+/// Superview container for platform views, to which sublayer transforms are applied.
+@interface FlutterPlatformViewContainer : NSView
+@end
+
+@implementation FlutterPlatformViewContainer
+
+- (BOOL)isFlipped {
+  // Flutter transforms assume a coordinate system with an upper-left corner origin, with y
+  // coordinate values increasing downwards.
```

Comment:
Maybe mention that the value also affects the layer.sublayerTransform rotation direction. Even though it kinda makes sense, I haven't found it in any AppKit/QuartzCore documentation.

Reply:
Done. Good call. Mentioned it affects view coords, transforms, and sublayerTransforms.